scss/styles.scss

```diff
@@ -141,3 +121,65 @@
   padding: 10px;
   text-transform: uppercase;
 }
+
+.front-grid {
+  display: grid;
+  grid-template-columns: repeat(3, max-content);
+  grid-template-areas:
+    'header header header'
+    'react angular vue'
+    'react elm polymer'
+}
+
+.tech.react  {
+  border-right: 1px solid $green;
+  grid-area: react;
+  justify-content: center;
+}
+
+.tech.angular  {
+  grid-area: angular;
+}
+
+.tech.vue  {
+  grid-area: vue;
+}
+
+.tech.elm  {
+  grid-area: elm;
+}
+
+.tech.polymer  {
+  grid-area: polymer;
+}
+
+.tech.polymer  {
+  grid-area: polymer;
+}
+
+.back-grid {
+  display: grid;
+  grid-template-columns: repeat(3, max-content);
+  grid-template-areas:
+    'header header header'
+    'ruby node asp'
+    'ruby django footer'
+}
+
+.tech.ruby  {
```

Comment:
we should add a padding-right to separate the content from the border here and in the react selectors